lib/plugin.js

```diff
@@ -175,6 +175,13 @@ module.exports = {
     ]
   },
   'processors': [
+    {
+      'name': 'variable',
+      'expr': /^--/im,
+      'action': function (prop, value) {
+        return { prop, value }
```

Comment:
@danfooo Please change this to ES5 syntax, just for the sake of code base consistency :)